ci/ci.sh

```diff
@@ -32,9 +32,13 @@ wget -q https://dl.k8s.io/release/v1.27.3/bin/linux/amd64/kubectl -O /home/runne
 chmod +x /home/runner/go/bin/kubectl
 }
 
-
 create_kind_cluster(){
-kind create cluster
+kubectl_version="v1.27.3"
```

Comment:
doesn't this mean that the kubernetes version is hardcoded to 1.27.3 ?

Reply:
well actually it's the default value, and when you call it with the "-v" flag, it then gets overwritten

Reply:
the version does come from the yaml CI